spec/teams_spec.rb

```diff
@@ -0,0 +1,28 @@
+require_relative 'spec_helper'
+require './lib/teams'
+
+RSpec.describe Teams do
+  describe '#initialize' do
+    it 'exists' do
+      team1 = Teams.new(1, 'Atlanta United')
+      expect(team1).to be_an_instance_of(Teams)
+    end
+
+    it 'has attributes' do
+      team1 = Teams.new(1, "Atlanta United")
+      expect(team1.team_id).to eq(1)
+      expect(team1.teamName).to eq("Atlanta United")
+    end
+  end
+
+  describe '#self.load_csv' do
+    it 'creates team from path and adds them to array through iteration' do
+      file_path = './spec/fixtures/fixture_teams.csv'
+      teams = Teams.load_csv(file_path)
+
```

Comment:
you could add an assertion line here that also checks the correct number are being created, this would add just a bit more roundness to your testing.